src/mixins/maskable.js

```diff
@@ -108,9 +109,9 @@ export default {
       this.$nextTick(() => {
         if (!this.$refs.input) return
 
-        this.$refs.input.value = this.maskText(this.lazyValue)
+        this.$refs.input.value = this.maskText(this.$refs.input.value)
 
```

Comment:
@nekosaur `maskText` takes in raw text, whereas `this.$refs.input.value` is already masked.